code/addons/storyshots/storyshots-core/src/frameworks/configure.ts

```diff
@@ -11,7 +11,7 @@ import type {
 import { toRequireContext } from '@storybook/core-webpack';
 import { normalizeStoriesEntry } from '@storybook/core-common';
 import registerRequireContextHook from '@storybook/babel-plugin-require-context-hook/register';
-import global from 'global';
+import { global } from '@storybook/global';
```

Comment:
Hmmm, why isn't line 80 erroring below?  I had to add a `@ts-expect-error` in my PR: https://github.com/storybookjs/storybook/pull/19386/files#diff-e76d868bbea9b46f39c61e743ff5bcc0d53f8c76e51c0ed1b7aef5e2fa1c3daaR78

Reply:
good question, I don't know.. it's `any`:

![Screenshot 2022-12-13 at 18 56 32](https://user-images.githubusercontent.com/3070389/207409117-45027d5c-252d-464b-9075-7979e6dd2573.png)

![Screenshot 2022-12-13 at 18 56 37](https://user-images.githubusercontent.com/3070389/207409112-0943d087-3d00-4e33-a728-5fdf15bfb8e1.png)